project-koku/koku
# [COST-4623] - Migration to remove old fields

## Jira Ticket

[COST-4623](https://issues.redhat.com/browse/COST-4623)

## Description

This change will remove the old redundant fields from the manifest and report status models.

## Testing

1. Checkout Branch
2. Restart Koku
3. make create/load test customer
4. everything should function correctly.

## Notes

REQUIRES: https://github.com/project-koku/koku/pull/4880
REQUIRES: https://github.com/project-koku/koku/pull/4912
REQUIRES: https://github.com/project-koku/koku/pull/4926
...
